Generated prettier commands don't run

<!--
NOTICE: Please use the bug report only for reporting bugs on the application itself.

For general bugs with Lightning Web Components Open Source visit the LWC repository on https://github.com/salesforce/lwc.
-->

### Summary

Generated prettier commands don't run.

### Steps To Reproduce:

1.  I'm on a Mac and using NPM.
1.  Create a project with `create-lwc-app`.
1.  run `npm run prettier`

### Expected result

Prettier should run successfully.

### Actual result

```
> conference-app@0.0.0 prettier /Users/areineke/git/conference-app
> prettier --write \"**/*.{css,html,js,json,md,ts,yaml,yml}"

sh: -c: line 0: unexpected EOF while looking for matching `"'
sh: -c: line 1: syntax error: unexpected end of file
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! conference-app@0.0.0 prettier: `prettier --write \"**/*.{css,html,js,json,md,ts,yaml,yml}"`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the conference-app@0.0.0 prettier script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/areineke/.npm/_logs/2020-01-22T23_44_30_657Z-debug.log
The terminal process terminated with exit code: 2
```

### Additional information

I'll send a PR that should fix this, though I haven't tested the PR yet.

--- a/packages/create-lwc-app/src/generators/createGenerator.ts
+++ b/packages/create-lwc-app/src/generators/createGenerator.ts
@@ -266,9 +266,9 @@ class CreateGenerator extends Generator {
             this.pjson.scripts.lint = 'eslint ./src/**/*.js'
         }
         this.pjson.scripts.prettier =
-            'prettier --write \\"**/*.{css,html,js,json,md,ts,yaml,yml}"'
+            'prettier --write \"**/*.{css,html,js,json,md,ts,yaml,yml}"'
         this.pjson.scripts['prettier:verify'] =
-            'prettier --list-different \\"**/*.{css,html,js,json,md,ts,yaml,yml}"'
+            'prettier --list-different \"**/*.{css,html,js,json,md,ts,yaml,yml}"'
         if (this.clientserver) {
             if (this.typescript) {
                 this.pjson.scripts.build =

